### ISSUE TITLE
First login - what credentials should I use?

### ISSUE THREAD

**Issue by roukydesbois** [YunoHost-Apps/calibre_ynh#7]
Hello!

I just installed calibre-web on my yunohost, and I can't log into it. I tried my yunohost credentials (but I saw that LDAP is not set up so it's normal), I tried the admin password that I set at installation with both the user I said was admin at the install, and with the admin user, and finally the default password (Admin123) as described in the "original" repo. So to summarize:
- [ynh_user]/[ynh_passwd]
- [ynh_calibre_admin_user]/[ynh_calibre_admin_password]
- admin/[ynh_calibre_admin_password]
- admin/Admin123

Am I missing something? Thanks!

**Comment by Krakinou** [comment]
Hello,
At install time, user Admin and related password admin123 are overwritten by the ones you set, so :
ynh_calibre_admin_user]/[ynh_calibre_admin_password]
Should be good.
Are you sure you're not making a typo in the ynh_calibre_admin_password?
Can you give me the output of 
```
sqlite3 /car/www/calibreweb/app.db ".dump"
```

**Comment by Krakinou** [comment]
OK.
In some cases, it seems that the install script do not succed in generating a password and writing it to the database.
You may regenerate it using following command:
```
sqlite3 /var/www/calibreweb/app.db "UPDATE user SET password='$(python /etc/yunohost/apps/calibreweb/conf/generate_password_hash.py yourincrediblepassword /var/www/calibreweb/vendor)' WHERE ID=1"
```
I'm searching the root cause.

**Comment by Krakinou** [comment]
Due to a bad variable handling, when the password contained a space or some other weird character, it was not correctly pass to the calibre-web database.
This is now corrected:  #8 
You may use the previous command line to correct the database or reinstall the package, no data shoudl be touched by the removing and reinstalling of the app.
Thanks for your feedback!
